spec/requests/waste_carriers_engine/renewal_start_forms_spec.rb

```diff
@@ -35,7 +35,11 @@ module WasteCarriersEngine
               end
 
               context "when the registration cannot be renewed" do
-                before(:each) { registration.metaData.expire! }
+                before(:each) do
+                  registration.metaData.expire!
```

Comment:
If the registration is not renewable once it's past the grace period (regardless of status), wonder if we could make it unrenewable with a single line that also persists the change: `registration.update_attributes(expires_on: Date.today + Rails.configuration.grace_window)`